files/collect_api_events.rb

```diff
@@ -48,10 +48,16 @@ def main(confdir, logpath, lockdir)
   orchestrator = PeEventForwarding::Orchestrator.new(settings['pe_console'], **client_options)
   activities = PeEventForwarding::Activity.new(settings['pe_console'], **client_options)
 
-  service_names = (settings['disable_rbac'] == 'false') ? PeEventForwarding::Activity::SERVICE_NAMES_WITH_RBAC : PeEventForwarding::Activity::SERVICE_NAMES_WITHOUT_RBAC
+  service_names = if !settings['skip_events'].nil?
```

Comment:
some places you use `unless` some you use `if !`. I don't think it matters which you go with but i would stick to one or the other

Reply:
Made changes to ensure consistency throughout the code 👍.